openreview/journal/process/recruit_process.py

```diff
@@ -46,6 +48,37 @@ def process(client, note, invitation):
 
 '''.format(ACTION_EDITOR_NAME=ACTION_EDITOR_NAME, SHORT_PHRASE=SHORT_PHRASE)
 
-            return client.post_message(subject, [user], message, parentGroup=ACTION_EDITOR_DECLINED_ID)
+            response =  client.post_message(subject, [user], message, parentGroup=ACTION_EDITOR_DECLINED_ID)
+
+        action = 'accepted' if response == 'Yes' else 'declined'
+
+        if JOURNAL_REQUEST_ID:
+            recruitment_notes = list(openreview.tools.iterget_notes(client, invitation=f'.*/Journal_Request.*/-/Reviewer_Recruitment_by_AE', replyto=JOURNAL_REQUEST_ID))
+            print(len(recruitment_notes), 'recruitment notes!')
+            for note in recruitment_notes:
+                invitee = note.content['invitee_email']['value'].strip()
+                id_or_email = user
+                if '~' in user:
+                    profile = openreview.tools.get_profile(user)
+                    id_or_email = profile.id
+                if invitee == id_or_email:
```

Comment:
don't you need to get the profile of the invitee too? 

AEs can invitee reviewers using email addresses, right?